packages/ui/molecules/Hero/Hero.twig

```diff
@@ -0,0 +1,143 @@
+#}
+
+{% set attributes =
+  merge_html_attributes(attr ?? null, { class: ['hero', 'w-screen', 'h-screen', 'relative'] })
+%}
+
+{# Default images attributes #}
+{% set image_attributes =
+  merge_html_attributes(
+    image_attr ?? null,
+    {
+      class: ['hero__image', 'object-cover', 'w-full', 'h-full', 'absolute', 'z-under'],
+      src: src,
+      alt: alt|default(''),
+      width: width|default(false),
+      height: height|default(false)
+    }
+  )
+%}
+
+{# Default content attributes #}
+{% set content_position = content_position|default('center') %}
+{% set content_attributes =
+  merge_html_attributes(
+    content_attr ?? null,
+    { class: 'hero-content' },
+    {
+      class: [
+        'flex flex-col justify-center h-full w-full space-y-8',
+        {
+          'items-start': content_position == 'left',
+          'items-center': content_position == 'center',
+          'items-end': content_position == 'right'
+        }
+      ]
+    }
+  )
+%}
+{% set surtitle_tag = title_tag|default('p') %}
+{% set surtitle_attributes =
+  merge_html_attributes(title_attr ?? null, { class: ['hero-content__surtitle', 'type-base'] })
+%}
+
+{% set title_tag = title_tag|default('p') %}
+{% set title_attributes =
+  merge_html_attributes(title_attr ?? null, { class: ['hero-content__title', 'type-h1'] })
+%}
+
+{% set description_tag = description_tag|default('p') %}
+{% set description_attributes =
+  merge_html_attributes(
+    description_attr ?? null,
+    { class: ['hero-content__description', 'type-base'] }
+  )
+%}
+
+{% set cta_attributes =
+  merge_html_attributes(cta_attr ?? null, { class: ['hero-content__cta', 'flex', 'px-6'] })
+%}
+
+<section {{ html_attributes(attributes) }}>
+  {% block background %}
+    {% if image %}
+      <div {{ html_attributes(image_attributes) }}>
+        {% include '@ui/atoms/Figure/Figure.twig' with image only %}
+      </div>
+    {% endif %}
+  {% endblock %}
+  {% block content %}
+    <div {{ html_attributes(content_attributes) }}>
+      {% block surtitle %}
+        {% if surtitle %}
+          {% html_element surtitle_tag with surtitle_attributes %}
+            {{ surtilte }}
+          {% end_html_element %}
+        {% endif %}
+      {% endblock %}
+      {% block title %}
+        {% if title %}
+          {% html_element title_tag with title_attributes %}
+            {{ title }}
+          {% end_html_element %}
+        {% endif %}
+      {% endblock %}
+      {% block description %}
+        {% if description %}
+          {% html_element description_tag with description_attributes %}
+            {{ description }}
+          {% end_html_element %}
+        {% endif %}
+      {% endblock %}
+      {% block ctas %}
+        {% if ctas %}
+          {% for cta in ctas %}
+            <div {{ html_attributes(cta_attributes) }}>
+              {% include '@ui/atoms/Button/Button.twig' with {
+                label: cta.cta_label,
+                url: cta.cta_url
+              } %}
```

Comment:
**praise:** We should directly pass the `cta` variable to the `include` here

This would allow for easier configuration of each CTA.

```suggestion
              {% include '@ui/atoms/Button/Button.twig' with cta only %}
```